DataParTemplate.m

```diff
@@ -236,6 +236,15 @@
 % 							   - 1 = enabled
 % 							   - 0 = disabled
 %
+% -----------------------------------------------------------------------------------------------------------------------------------------------------
+% VISUALIZATION PARAMETERS
+% -----------------------------------------------------------------------------------------------------------------------------------------------------
+% x.bVisualQCCBFvsGMWMTemplate - if == 1, uses the template GM and WM to make visualization overlays with CBF and other derived maps
+%                              if == 0, uses the individual GM and WM maps of the subject to make visualizations
+%                              (OPTIONAL, DEFAULT = 0)
+% x.bVisualQCCBFvsGMWMContour -  if == 1, then produces the WM and GM as a contour in the overlay with CBF and derived maps, not a full ROI - 1 pixel thickness (contour calculated per slice basis).
```

Comment:
GM and WM is switched compared to above (WM and GM). Perhaps better to put these booleans in `x.vis.` or `x.stats.`